Assets/HoloToolkit/Common/Scripts/WorldAnchorManager.cs

```diff
@@ -229,6 +229,80 @@ public string AttachAnchor(GameObject gameObjectToAnchor, string anchorName = nu
 #endif
         }
 
+        /// <summary>
+        /// Attaches an anchor immediately to the specified GameObject.  
+        /// If the anchor store has an anchor with the specified name it will load the anchor, 
+        /// otherwise a new anchor will be saved under the specified name.  
+        /// If no anchor name is provided, the name of the anchor will be the same as the GameObject.
+        /// </summary>
+        /// <param name="gameObjectToAnchor">The GameObject to attach the anchor to.</param>
+        /// <param name="anchorName">Name of the anchor.  If none provided, the name of the GameObject will be used.</param>
+        /// <returns>The the newly attached anchor.</returns>
+        public UnityEngine.XR.WSA.WorldAnchor AttachAnchorImmediate(GameObject gameObjectToAnchor, string anchorName = null)
```

Comment:
Doesn't compile for 5.6 & 2017.1 and for builds that target a different platform than WSA

Reply:
Should we make this change in a separate PR?
This seems to be adding functionality.

Reply:
Will do.